server/image_pull.go

```diff
@@ -272,7 +272,7 @@ func tryIncrementImagePullFailureMetric(ctx context.Context, img string, err err
 	}
 }
 
-func decodeDockerAuth(s string) (user, password string, err error) {
+func decodeDockerAuth(s string) (user, password string, errRet error) {
```

Comment:
I don't think errRet needs to be defined here.

Reply:
in this case, it seems the return value is being used as documentation: which string is which. that's one of the cases I've decided to leave